apps/web/screens/Layouts/SidebarLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC, ReactNode } from 'react'
+import {
+  Box,
+  GridContainer,
+  GridRow,
+  GridColumn,
+  Hidden,
+} from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+
+import * as styles from './SidebarLayout.treat'
+import cn from 'classnames'
+type Breakpoint = keyof typeof theme['breakpoints']
+
+interface SidebarLayoutProps {
+  sidebarContent: ReactNode
+  absolute?: boolean
+  hiddenBelow?: Exclude<Breakpoint, 'xs'>
+}
+
+export const SidebarLayout: FC<SidebarLayoutProps> = ({
+  sidebarContent,
+  absolute,
+  hiddenBelow,
+  children,
+}) => (
+  <GridContainer position="none">
+    <Box
+      display="flex"
+      flexDirection="row"
+      height="full"
+      paddingBottom={6}
+      style={{
+        position: absolute ? undefined : 'relative',
+      }}
```

Comment:
Is it possible to use `Box` props here instead of style?
```suggestion
      position={absolute ? undefined : 'relative'}
```